fast64_internal/utility.py

```diff
@@ -1322,3 +1375,27 @@ def rotate_quat_blender_to_n64(rotation: mathutils.Quaternion):
 
 def all_values_equal_x(vals: Iterable, test):
     return len(set(vals) - set([test])) == 0
+
+
+def get_blender_to_game_scale(context):
```

Comment:
Another spot that could be included in the README for adding new game-specific render settings?